content/en/docs/concepts/overview/working-with-objects/field-selectors.md

```diff
@@ -32,6 +32,18 @@ kubectl get ingress --field-selector foo.bar=baz
 Error from server (BadRequest): Unable to find "ingresses" that match label selector "", field selector "foo.bar=baz": "foo.bar" is not a known field selector: only "metadata.name", "metadata.namespace"
 ```
 
+### List of supported fields
```

Comment:
I think this list is missing CertificateSigningRequest `spec.signerName`

Otherwise, the list looks technically accurate

Reply:
I added the missing field selector